tests/quantization/bnb/test_mixed_int8.py

```diff
@@ -509,6 +511,29 @@ def test_quality(self):
         max_diff = numpy_cosine_similarity_distance(expected_slice, out_slice)
         self.assertTrue(max_diff < 1e-3)
 
+    @require_peft_version_greater("0.14.0")
+    def test_lora_loading_works(self):
```

Comment:
```suggestion
    def test_lora_loading(self):
```

Reply:
Can we add 4bit testing too?

Reply:
Contingent on https://github.com/huggingface/diffusers/pull/10578/ and is being added in the same PR.